source/unified-test-format/unified-test-format.rst

```diff
@@ -462,6 +477,90 @@ The structure of this object is as follows:
 
     Test files SHOULD NOT use this option unless one or more command monitoring
     events are specified in `observeEvents <entity_client_observeEvents_>`_.
+    
+  - ``storeEventsAsEntities``: Optional map of event names to entity names.
+    If provided the test runner MUST:
+    
+      - For each entity name, create the respective entity with a type of
```

Comment:
Can we omit the bullets here and just rely on paragraphs? I think that'd help readability. See `useMultipleMongoses` for an example.

Reply:
Done.